### DISCUSSION TITLE
Is 1Wh/imp possible?

### DISCUSSION THREAD

**Discussion by slepax** [klaasnicolaas/home-assistant-glow#239] [Q&A] [answered]
The meter we have installed shows 1Wh/imp next to the pulse light.

Is this possible?

That's quite a fair bit down compared to the more readily available kWh/imp most people have?

![image](https://user-images.githubusercontent.com/13784746/204836467-bc2884af-f628-46ff-ac0c-59b56aa82bd7.png)

**Comment by klaasnicolaas** [comment]
I doubt whether this will really work, but I am curious what the behavior of the LED is. Because I think it will flash very often and therefore quite an ESP may be overloaded 🤔

**Comment by johankvdbp** [comment, marked as answer]
Hi. 1KWh/1000imp = 1Wh/imp.  1KWh = 1000Wh

**Reply by klaasnicolaas** [reply]
So he can use the value 1000, do I understand that correctly?  😅

**Reply by slepax** [reply]
Thanks, that makes sense. I overlooked the "1000" above the imp/kWh in the example photo.

**Reply by johankvdbp** [reply]
Yes 1000 can still be used. 
Thanks for your hard work on this project mine has been operating for some time now with a modified moisture sensor/ldr setup. Could not wait for the correct sensor and had some unused spares.
